cmd/normalizer/main.go

```diff
@@ -8,14 +8,17 @@ import (
 	"syscall"
 
 	"github.com/go-kit/kit/log"
+	kitprometheus "github.com/go-kit/kit/metrics/prometheus"
 	"github.com/mainflux/mainflux"
 	"github.com/mainflux/mainflux/normalizer"
+	"github.com/mainflux/mainflux/normalizer/api"
 	nats "github.com/nats-io/go-nats"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
 )
 
 const (
 	defNatsURL string = nats.DefaultURL
-	defPort    string = "8180"
+	defPort    string = "8900"
```

Comment:
Why? I thought that @mijicd fixed all services to same port, demanding env variable to define the desired port.

Reply:
It was easier for me to run that way, since I run all services locally, I don't use Docker for development. I accidentally committed that change.